src/types.ts

```diff
@@ -521,14 +522,11 @@ export enum VALIDATION_INPUT_TYPES {
   ARWEAVE_ID = 'Is valid Arweave Transaction (TX) ID.',
   ARWEAVE_ADDRESS = 'Is likely an Arweave wallet address.',
   AO_ADDRESS = 'Is a valid AO Address.',
-  ARNS_NAME = 'ARNS Name.',
+
   UNDERNAME = 'Is a valid Undername.',
-  ANT_CONTRACT_ID = 'Is a valid Arweave Name Token (ANT).',
   // unfortunately we cannot use computed values in enums, so be careful if we ever modify this number
-  TRANSACTION_CONFIRMATIONS = `Has sufficient confirmations (50+).`,
-  VALID_TTL = `Minimum ttl allowed is 900 and Maximum ttl allowed is 2,592,000.`,
-  EMAIL = `Is a valid email`,
-  SMARTWEAVE_CONTRACT = `Is a SmartWeave Contract`,
+  VALID_TTL = `Minimum ttl allowed is ${MIN_TTL_SECONDS} and Maximum ttl allowed is ${MAX_TTL_SECONDS}.`,
```

Comment:
```suggestion
  VALID_TTL = `TTL must between ${MIN_TTL_SECONDS} and ${MAX_TTL_SECONDS} seconds`,
```

Reply:
can also make it human friendly error message so it reads as 

`TTL must be between 1 minute (60 seconds) and 1 day (86400 seconds)`